### PR TITLE
Changed azure cli version extraction

### PR DESCRIPTION
**Pull request by hiyadav**
Fixing #774

### CODE REVIEW

File: images/win/scripts/Installers/Validate-AzureCli.ps1 [hunk 1]
```diff
@@ -14,17 +14,13 @@ else
     exit 1
 }
 
-
-if( $($(az --version) | out-string)  -match  'azure-cli \((?<version>.*)\)' )
-{
-   $azureCliVersion = $Matches.version
-}
+$azureCliVersion = az -v | findstr azure-cli
```
Comment (sachinma) [inline images/win/scripts/Installers/Validate-AzureCli.ps1]: whats the new syntax of the version?

Comment (hiyadav) [reply images/win/scripts/Installers/Validate-AzureCli.ps1]: The new syntax doesn't have braces before the version,
thus it is show like: 
**_az-cli       2.0.63_**

Comment (sachinma) [reply images/win/scripts/Installers/Validate-AzureCli.ps1]: Should we just remove the braces from the previous regex in that case

Comment (hiyadav) [reply images/win/scripts/Installers/Validate-AzureCli.ps1]: The previous regex has not been working from long, whereas this one is producing right results. Hence, I preferred using this one over altering the existing one. The new one is being used in other validation helper files.

Comment (sachinma) [reply images/win/scripts/Installers/Validate-AzureCli.ps1]: Only thing is the extra az-cli in the version field; should we remove that? I think that is probably fine; but see if it is easy to remove that part

Comment (hiyadav) [reply images/win/scripts/Installers/Validate-AzureCli.ps1]: Okay, I'll remove the extra _azure-cli_

Comment (hiyadav) [reply images/win/scripts/Installers/Validate-AzureCli.ps1]: done

**Pull request merged**